\documentclass[11pt]{article}
\usepackage{xcolor}
\usepackage[english]{babel}
\usepackage{fancyhdr}
%\usepackage{tabularx}
%\usepackage{tablestyles}
\usepackage[hidelinks=true]{hyperref}

% Reverse Numbering
\usepackage{etaremune}

% Set 1-inch margins.
\usepackage[margin=1in]{geometry}

% Headers and Footer
\pagestyle{fancy}
\fancyhf{}
\rhead{\textrm{\textbf{William E.\ Fondrie}}}
\lhead{\textrm{\textbf{Curriculum Vitae}}}
\rfoot{\thepage}
\renewcommand{\headrulewidth}{0pt}

\fancypagestyle{plain}{
  \fancyhf{}
  \rfoot{\thepage}
  \renewcommand{\headrulewidth}{0pt}
}

% Use Roboto font
\usepackage[rm]{roboto}
\usepackage[T1]{fontenc}
\renewcommand{\familydefault}{\sfdefault}

% Skip a line between paragraphs.
\setlength{\parskip}{1.5ex}

% Turn off indenting.
\setlength{\parindent}{0in}

% Define the section command.
\newcommand{\mysection}[1]{\vspace{1ex \textbf{\large \textrm{#1}} \quad
    \hrulefill}}
\newcommand{\mysubsection}[1]{\vspace{1ex \textbf{\textrm{#1}}}}

% Set the numbering style.
\renewcommand{\labelenumi}{(\arabic{enumi})}

% Make the \footnote command print symbols (*, etc).
\renewcommand{\thefootnote}{\fnsymbol{footnote}}

% Smarter version of href.
\newcommand{\myref}[1]{\href{#1}{\url{#1}}}

% Table sizes
\newlength{\leftcol}
\setlength{\leftcol}{0.6in}
\newlength{\rightcol}
\setlength{\rightcol}{5.25in minus \leftcol}
\newcommand{\tdim}{p{\leftcol}p{\rightcol}}
\renewcommand{\arraystretch}{1.25} % make rows taller

% No hyphenation
\pretolerance=10000
\raggedright

% colors
\definecolor{gray}{HTML}{404040}
\color{gray}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{document}
\thispagestyle{plain}
\begin{center}
  \textbf{\Large \textrm{William E Fondrie, PhD}} \\
  Talus Bioscience \\
  wfondrie@talus.bio \\
  % {phone #}\\
  \href{https://willfondrie.com}{willfondrie.com}\\
  Updated \today.
\end{center}


\mysection{Education}
\begin{tabular}{\tdim}
  % 9/2013--06/2018
  2013--18 & \textbf{PhD in Molecular Medicine}
             \newline University of Maryland, Baltimore
             \newline Advisors: David Goodlett, PhD and Dudley Strickland, PhD\\
  % 9/2009--05/2013
  2009--13 & \textbf{BS in Chemistry}
             \newline University of North Carolina at Chapel Hill\\
\end{tabular}

\mysection{Employment and Professional Appointments}

\begin{tabular}{\tdim}
  2021--   & \textbf{Senior Data Scientist}, Talus Bioscience\\
  2019--21 & \textbf{University of Washington Data Science Postdoctoral Fellow}
             \newline UW eScience Institute\\
  2018--21 & \textbf{Postdoctoral Fellow}, University of Washington
             \newline Advisor: William Noble, PhD\\
  2013--18 & \textbf{Graduate Research Assistant}, University of Maryland,
             Baltimore
             \newline Advisors: David Goodlett, PhD and Dudley Strickland, PhD\\
  2012--13 & \textbf{Undergraduate Research Assistant}, University of North Carolina at
             Chapel Hill
             \newline Advisor: John Papanikolas, PhD\\
\end{tabular}

\mysection{Awards and Honors}
\begin{tabular}{\tdim}
  %2021     & \textbf{UW eScience Postdoctoral Fellow Research Funds} \\
  2021     & \textbf{Rising Stars in Proteomics and Metabolomics: 40 under 40},
             \newline \textit{Journal of Proteome Research} \\
  2020     & \textbf{Emerging Talent: Academia}, American Society for Mass Spectrometry\\
  2019--21 & \textbf{Ruth L Kirschstein Institutional National Research Service Award}
             \newline NIH T32HG000035, Postdoctoral Trainee\\
  2017     & \textbf{Travel Fellowship to the May Institute on Computation and Statistics
             for\newline Mass Spectrometry and Proteomics}\\
  2017--18 & \textbf{Ruth L Kirschstein Individual National Research Service Award}
             \newline NIH F31CA213815, PI\\
             % \newline \textit{Impact Score: 25 (16\%)}\\
  2016--17 & \textbf{Ruth L Kirschstein Institutional National Research Service Award}
             \newline NIH T32HL007698, Predoctoral Trainee\\
  2012     & \textbf{Markham Summer Undergraduate Research Award}\\
  2009     & \textbf{Central Carolina's chapter of Phi Beta Kappa Scholarship}\\
\end{tabular}

\newpage
%\mysection{Preprints}
%\begin{etaremune}
%\end{etaremune}

\mysection{Peer-Reviewed Publications}
{\small (* indicates equal contributions)}

\begin{etaremune}
  \item \textbf{Fondrie WE}, Bittreieux W, Noble WS. (2021) ppx: Programmatic
  Access to Proteomics Data Repositories. \textit{J Proteome Res}.
  3;20(9):4621-4624. PMID: 34342226

  \item Mudge MC, Nunn BL, Firth E, Ewert M, Hales K, \textbf{Fondrie WE},
  Noble WS, Toner J, Light B, Junge KA. (2021) Subzero, saline incubations of
  Colwellia psychrerythraea reveal strategies and biomarkers for sustained life
  in extreme icy environments. \textit{Environ Microbiol}. 23(7):3840-3866.
  PMID: 33760340

  \item \textbf{Fondrie, WE}, Noble WS. (2021) mokapot: Fast and flexible
  semisupervised learning for peptide detection. \textit{J Proteome Res}.
  2;20(4):1966-1971. PMID: 33596079.

  \item \textbf{Fondrie WE}, Noble WS. (2020) Machine learning strategy that
  leverages large datasets to boost statistical power in small-scale
  experiments. \textit{J Proteome Res} 6;19(3):1267--1274. PMID: 32009418.
  
  \item Liang T, Leung LM, Opene B, \textbf{Fondrie WE}, Lee YI, Chandler CE,
  Yoon SH, Doi Y, Ernst RK, Goodlett DR. (2019) Rapid microbial identification
  and antibiotic resistance detection by mass spectrometric analysis of
  membrane lipids. \textit{Anal Chem} 91(2):1286--1294. PMID: 30571097.

  \item \textbf{Fondrie WE}, Liang T, Oyler BL, Leung LM, Ernst RK, Strickland
  DK, Goodlett DR. (2018) Pathogen Identification Direct From Polymicrobial
  Specimens Using Membrane Glycolipids. \textit{Sci Rep} 8(1):15857. PMID:
  30367087.
    
  \item Liang T, Schneider T, Yoon SH, Oyler BL, Leung LM, \textbf{Fondrie WE},
  Yen G, Huang Y, Ernst RK, Nilsson E, Goodlett DR. (2018) Optimized surface
  acoustic wave nebulization facilitates bacterial phenotyping. \textit{Int J
    Mass Spectrom} 427:65--72.
    
  \item Au DT, Arai AL, \textbf{Fondrie WE}, Muratoglu SC, Strickland DK.
  (2018) Role of the LDL Receptor-Related Protein 1 in Regulating Protease
  Activity and Signaling Pathways in the Vasculature. \textit{Curr Drug
    Targets} 19(11):1276--1288. PMID: 29749311.

  \item Au DT, Ying Z, Hern√°ndez-Ochoa EO, \textbf{Fondrie WE}, Hampton B,
  Migliorini M, Galisteo R, Schneider MF, Daugherty A, Rateri DL, Strickland
  DK, Muratoglu SC. (2018) LRP1 (Low-Density Lipoprotein Receptor-Related
  Protein 1) Regulates Smooth Muscle Contractility by Modulating Ca2+ Signaling
  and Expression of Cytoskeleton-Related Proteins. \textit{Arterioscler Thromb
    Vasc Biol} 38(11):2651--2664. PMID: 30354243.
    
  \item Khan MM, Tran BQ, Jang Y, Park S, \textbf{Fondrie WE}, Chowdhury K,
  Yoon SH, Goodlett DR, Chae S, Chae H, Seo S, Goo YA. (2017) Assessment of the
  therapeutic potential of persimmon leaf extract on prediabetic subjects. {\it
    Mol Cells} 40(7):466. PMID: 28743946.
    
  \item Leung LM, \textbf{Fondrie WE}, Doi Y, Johnson JK, Strickland DK, Ernst
  RK, Goodlett DR. (2017) Identification of the ESKAPE pathogens by mass
  spectrometric analysis of microbial membrane glycolipids. {\it Sci Rep}
  7(1):6403. PMID: 28743946.

  \item Clark DJ, \textbf{Fondrie WE}, Liao Z, Yang AJ, Mao L. (2016) Triple
  SILAC quantitative proteomic analysis reveals differential abundance of cell
  signaling proteins between normal and lung cancer-derived exosomes. {\it J
    Proteomics} 133:161--169. PMID: 26739763.

  \item Clark DJ*, \textbf{Fondrie WE*}, Liao Z, Hanson PI, Fulton A, Mao L,
  Yang AJ. (2015) Redefining the breast cancer exosome proteome by tandem mass
  tag quantitative proteomics and multivariate cluster analysis. \textit{Anal
    Chem} 87(20):10462--10469. PMID: 26378940.
    
  \item Ma D, Bettis SE, Hanson K, Minakova M, Alibabaei L, \textbf{Fondrie W},
  Ryan DM, Papoian GA, Meyer TJ, Waters ML, Papanikolas JM. (2013) Interfacial
  energy conversion in Ru(II) polypyridyl-derivatized oligoproline assemblies
  on TiO2. \textit{J Am Chem Soc} 135(14):5250--5253. PMID: 23514453.
\end{etaremune}

\mysection{Additional Publications}
\begin{etaremune}
  \item \textbf{Fondrie WE}. Biological Insight from Mass Spectrometry Through
  Novel Computational Approaches. Ph.D. dissertation. University of Maryland,
  Baltimore. June, 2018. Advisors: Dudley K. Strickland, Ph.D. and David R.
  Goodlett, Ph.D.
\end{etaremune}

\mysection{Patents}
\begin{etaremune}
  \item Goodlett DR, Ernst RK, Liang T, \textbf{Fondrie WE}, Nilsson E. (2018)
  Methods for Lipid Extraction and Identification of Microbes Using Same Via
  Mass Spectrometry. US Patent Application 2017066342. Filed 12/14/2017. {\it
    Patent Pending.}
\end{etaremune}


\mysection{Scientific Software Contributions}
\begin{etaremune}
  \item \textbf{mokapot}. A Python package that enables the fast and flexible
  application of semi-supervised learning to detect peptides in proteomics
  experiments. \\
  Homepage: \myref{https://github.com/wfondrie/mokapot} \\
  Roles: creator and maintainer

  \item \textbf{ppx}. A Python package for programmatic access to proteomics
  data repositories. \\
  Homepage: \myref{https://github.com/wfondrie/ppx} \\
  Roles: creator and maintainer

  \item \textbf{depthcharge}. A deep learning toolkit for proteomics data. \\
  Homepage: \myref{https://github.com/wfondrie/depthcharge} \\
  Roles: creator and maintainer

  \item \textbf{Crux}. A mass spectrometry toolkit for proteomics data. \\
  Homepage: \myref{http://crux.ms} \\
  Roles: contributor

  \item \textbf{Percolator}. Semi-supervised learning for peptide
  identification from shotgun proteomics datasets. \\
  Homepage: \myref{http://percolator.ms} \\
  Roles: contributor

  \item \textbf{dask-ml}. A library for distributed and parallel machine
  learning using dask. \\
  Homepage: \myref{https://ml.dask.org} \\
  Roles: contributor

  \item \textbf{Kojak}. A search engine for cross-linking proteomics. \\
  Homepage: \myref{http://www.kojak-ms.org} \\
  Roles: contributor

  \item \textbf{MSToolkit}. A library for interacting with proteomics data formats. \\
  Homepage: \myref{https://github.com/mhoopmann/mstoolkit} \\
  Roles: contributor

  \item \textbf{ANN-SoLo}. A spectral library search engine for fast and
  accurate open modification searching. \\
  Homepage: \myref{https://github.com/bittremieux/ANN-SoLo} \\
  Roles: contributor
   
  \item \textbf{spectrum\_utils}. A Python package for efficient MS/MS spectrum
  processing and visualization. \\
  Homepage: \myref{https://github.com/bittremieux/spectrum_utils} \\
  Roles: contributor.

  \item \textbf{xComb}. A cross-linked peptide database approach to
  protein-protein interaction analysis. \\
  Homepage: \myref{https://github.com/GoodlettLab/xComb} \\
  Roles: maintainer and contributor

\end{etaremune}

\mysection{Talks}
\begin{etaremune}
  \item \textbf{Fondrie WE}, Romero E, Noble WS. Learning Compact
  Representations of High-Resolution Tandem Mass Spectra. 69th American Society
  for Mass Spectrometry Annual Conference on Mass Spectrometry and Allied
  Topics, October 31--November 4, 2021. Philadelphia, PA. \textit{Presented by
    Noble WS due to COVID travel risks.}

  \item \textbf{Fondrie WE}, Romero E, Noble WS. Learning Compact
  Representations of High-Resolution Tandem Mass Spectra Without Supervision.
  Intelligent Systems for Molecular Biology, July 25--30, 2021. Online.

  \item \textbf{Fondrie WE}. Deep Learning to Enable Proteomics at Scale.
  American Society for Mass Spectrometry Emerging Talent: Academia webinar
  series, November 10--12, 2020. Online.
  
  \item \textbf{Fondrie WE}. Learning from Mass Spectra. Genome Sciences
  Sciences Seminar, August 12, 2020. University of Washington Department of
  Genome Sciences, Seattle, WA
  
  \item \textbf{Fondrie WE}, Leung LM, Strickland DK, Ernst RK, Goodlett DR.
  Detecting antibiotic resistance by MALDI-TOF analysis of bacterial membrane
  glycolipids. 65th American Society for Mass Spectrometry Annual Conference on
  Mass Spectrometry and Allied Topics, June 4--8, 2017. Indianapolis, IN.
    
  \item \textbf{Fondrie WE}, Muratoglu SC, Hampton B, Migliorini M, Galisteo R,
  Strickland DK. LRP1 modulates TGF-$\beta$ signaling in the descending
  thoracic aorta. Molecular Medicine Research Retreat, October 6, 2016.
  Baltimore, MD.
\end{etaremune}


\mysection{Posters}
\begin{etaremune}
  \item \textbf{Fondrie, WE}, Noble WS. Mokapot: Fast and Flexible
  Semi-Supervised Learning for Peptide Detection. US Human Proteome
  Organization 17th Annual Conference. March 4--12, 2021. Virtural.

  \item \textbf{Fondrie, WE}, Noble WS. Mokapot: Fast and Flexible
  Semi-Supervised Learning for Peptide Detection. 28th Conference on
  Intelligent Systems for Molecular Biology, July 13--16, 2020. Montreal, QC,
  Canada. \textit{Moved online due to COVID-19.}
  
  \item \textbf{Fondrie, WE}, Noble WS. Boosting statistical power in
  small-scale experiments with Percolator. US Human Proteome Organization 16th
  Annual Conference. March 8--11, 2020. Seattle, WA \textit{Moved online due to
    COVID-19.}
  
  \item \textbf{Fondrie WE}, Noble WS. Robust Cross-Linked Peptide Detection
  Using Pretrained Neural Networks. 67th American Society for Mass Spectrometry
  Annual Conference on Mass Spectrometry and Allied Topics, June 1--6, 2019.
  Atlanta, GA.

  \item \textbf{Fondrie WE}, Hampton B, Muratoglu SC, Goodlett DR, Strickland
  DK. Detecting LRP1B protein interactions in glioma. UMB Cancer Biology
  Research Retreat, June 13, 2017. Baltimore, MD.

  \item \textbf{Fondrie WE}, Muratoglu SC, Hampton B, Migliorini M, Galisteo R,
  Strickland DK. LRP1 modulates TGF-$\beta$ signaling in the descending
  thoracic aorta. GenTAC Thoracic Aortic Summit, September 22--23, 2016.
  Washington, DC.

  \item \textbf{Fondrie WE}, Hampton B, Muratoglu SC, Goodlett DR, Strickland
  DK. Defining a mechanism of LRP1B tumor suppression in glioblastoma. UMB
  Cancer Biology Research Retreat, May 23, 2016. Baltimore, MD.

  \item \textbf{Fondrie WE}, Clark DJ, Catania SM, Goo YA, Strickland DK,
  Goodlett DR. Investigating the regulated intramembrane proteolysis of LRP1B
  in glioblastoma progression through a proteogenomic approach. Mass
  Spectrometry in Biotechnology and Medicine, July 5--11, 2015. Dubrovnik,
  Croatia.

  \item \textbf{Fondrie WE}, Clark DJ, Liao Z, Chen Y, Yang AJ. Novel
  identification of JAK1/STAT signaling proteins in breast cancer exosomes
  through shotgun proteomic analysis using multiple protein database search
  algorithms. UMB Cancer Biology Research Retreat, June 9, 2014. Baltimore, MD.
    
  \item \textbf{Fondrie WE}, Bettis S, Ma D, Minakova M, Wilger D, Papoian G,
  Waters M, Papanikolas J. Flexibility matters: The role of scaffold tethers in
  Ru(II) and Os(II) chromophore separation. Southeastern Regional Meeting of
  the American Chemical Society, November 14--17, 2012. Raleigh, NC.
\end{etaremune}


\mysection{Teaching}
\mysubsection{Training}
\begin{tabular}{\tdim}
  2020 & The Carpentries Instructor Training
         \newline \myref{https://carpentries.org/} \\
\end{tabular}

\mysubsection{Experience}
\begin{tabular}{\tdim}
  2021 & UW Fall Quarter Software Carpentry Workshop Instructor
         \newline \myref{https://uwescience.github.io/2021-10-11-uw-online/}\\
  2021 & UW Spring Quarter Software Carpentry Workshop Instructor
         \newline \myref{https://uwescience.github.io/2021-05-10-uw-online/}\\
  2021 & Lecture for the ``Statistics for Proteomics'' video series
         \newline \textit{Course website}: \myref{https://slavovlab.net/stats}
         \newline \textit{Lecture}: \myref{https://youtu.be/up7Bl0eeedk}\\
  2021 & UW Winter Quarter Software Carpentry Workshop Instructor
         \newline \myref{https://uwescience.github.io/2021-01-11-uw-online/} \\

\end{tabular}
  
\mysection{Mentoring}
\begin{tabular}{\tdim}
  \raggedright
  2021--    & \textbf{Lincoln Harris}, PhD student\newline Department of Genome Sciences, University of Washington\\
  2020      & \textbf{Elena Romero}, PhD student\newline Department of Genome Sciences, University of Washington\\
  2020--21  & \textbf{Donovan See}, Undergraduate student\newline University of Washington\\
  2020--21  & \textbf{Lilian Heil}, PhD student\newline Department of Genome Sciences, University of Washington\\
  2019--    & \textbf{Kianna Hales}, PhD student\newline Department of Genome Sciences, University of Washington\\

\end{tabular}

\mysection{Professional Service}

\mysubsection{Editorial Boards}
\begin{tabular}{\tdim}
  2021--23 & \textit{Journal of Proteome Research}. Editorial Advisory Board. \\
\end{tabular}

\mysubsection{Conference Organization}
\begin{tabular}{\tdim}
  2021 & Intelligent Systems for Molecular Biology CompMS COSI. \textit{Program Committee Member}. \\
  2020 & Workshop on High Performance Computing, Big Data Analytics and
         Integration for Multi-Omics Biomedical Data (HPC-BOD 2020). \textit{Program
         Committee Member}.\\
\end{tabular}

\mysubsection{Professional Memberships}
\begin{tabular}{\tdim}
  2021--     & American Chemical Society (ACS) \\
  2021--     & Human Proteome Organization (HUPO) \\
  2020--     & United States Human Proteome Organization (US HUPO) \\
  2013--     & American Society for Mass Spectrometry (ASMS) \\
\end{tabular}

\mysubsection{Departmental Service}
\begin{tabular}{\tdim}
  2019--21   & UW Genome Sciences Art Committee \\
  2018--21   & Postdoctoral representative in UW Genome Sciences\\
  2015--17   & UMB Molecular Medicine Event Planning Committee \\
  2017       & UMB Cancer Biology Research Retreat Organizing Committee \\
  2016       & UMB Grollman Lecture Organizing Committee \\
\end{tabular}

\textbf{Journal Referee: }\textit{Bioinformatics,
  Journal of Proteome Research, Scientific Reports}\\
\textbf{Conference Referee: }{Intelligent Systems for Molecular Biology 2020,
  2021; Research in Computational Molecular Biology 2021}

\mysection{Scientific Blog Posts}
\begin{etaremune}
  \item \textbf{Fondrie WE}. (2019) Using Static Models in Percolator. \\
  \myref{https://willfondrie.com/2019/11/using-static-models-in-percolator/}

  \item \textbf{Fondrie WE}. (2019) An Intuitive Look at the xcorr Score
  Function in Proteomics. \\
  \myref{https://willfondrie.com/2019/02/an-intuitive-look-at-the-xcorr-score-function-in-proteomics/}

\end{etaremune}


\mysection{References}

\textbf{Alexander Federation}\\
Co-founder and CEO, Talus Bioscience \\
afederation@talus.bio

\textbf{Lindsay Pino}\\
Co-founder and CTO, Talus Bioscience \\
lpino@talus.bio

\textbf{William Noble} \\
Department of Genome Sciences, University of Washington\\
wnoble@uw.edu

\textbf{Michael MacCoss} \\
Department of Genome Sciences, University of Washington\\
maccoss@uw.edu

\textbf{Dudley Strickland} \\
Center for Vascular and Inflammatory Diseases, University of Maryland
Baltimore\\
dstrickland@som.umaryland.edu

\textbf{David Goodlett} \\
Genome BC Proteome Centre, University of Victoria\\
goodlett@uvic.ca

\end{document}
